core/src/main/java/by/ilyin/core/controller/CargoTrackingCoreController.java

```diff
@@ -0,0 +1,25 @@
+package by.ilyin.core.controller;
+
+import by.ilyin.core.entity.dto.AppInfoDTO;
+import by.ilyin.core.service.AppInfoService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/api")
+public class CargoTrackingCoreController {
```

Comment:
Better AppInfoCoreController

Reply:
@Egor-Duvalov Please do not suggest some better name - just point that name should be better.
@YauhenIlyin FYI